subgraphs/_reference_/src/sdk/protocols/bridge/account.ts

```diff
@@ -267,6 +268,83 @@ export class Account {
     return transfer;
   }
 
+  /**
+   * Creates a BridgeMessage entity for a message arriving to this chain
+   *
+   * @param srcChainId the source chain id
+   * @param source The account sending the message.
```

Comment:
I think the `source` should be `Address | null`, since it might not be possible to always know where it is coming from? WDYT?

Reply:
Since both `from` and `to` are required fields for [BridgeMessage entity](https://github.com/messari/subgraphs/blob/9be3f84272faf4e0d397e4e36bb25267ecd06be2/schema-bridge.graphql#L1224), I think it makes sense to let the subgraph developer choose a value for `source` (zero address or receiver, or else), instead of the sdk making assumptions. What do you think? @jaimehgb

Reply:
Oh, we can change the schema too :D 

But zero address for whenever is unknown might work too.

Reply:
I slightly prefer to keep the current implementation and let subgraph developer choose the value for unavailable information.